src/test/scala/com/mpc/scalats/core/CompilerSpec.scala

```diff
@@ -1,15 +1,14 @@
 package com.mpc.scalats.core
 
-import scala.collection.immutable.ListSet
-
 import com.mpc.scalats.configuration.Config
+import com.mpc.scalats.core.ScalaParserResults._
+import com.mpc.scalats.core.TypeScriptModel._
+import org.scalatest.flatspec.AnyFlatSpec
+import org.scalatest.matchers.should.Matchers
 
-import org.scalatest.{ FlatSpec, Matchers }
-
-import TypeScriptModel._
-import ScalaParserResults._
+import scala.collection.immutable.ListSet
 
-final class CompilerSpec extends FlatSpec with Matchers {
+final class CompilerSpec extends AnyFlatSpec with Matchers {
```

Comment:
Why?

Reply:
Due to version bump of scalatest to 3.10.0